src/abstractdataframe/abstractdataframe.jl

```diff
@@ -375,29 +375,30 @@ describe(df, stats = [:min, :max])
 ```
 
 """
-function StatsBase.describe(df::AbstractDataFrame; stats::Union{Symbol,AbstractVector{Symbol}} =
-                            [:mean, :min, :median, :max, :nunique, :nmissing, :eltype])
-    # Check that people don't specify the wrong fields.
-    allowed_fields = [:mean, :std, :min, :q25, :median, :q75,
-                      :max, :nunique, :nmissing, :first, :last, :eltype]
-    if stats == :all
-        stats = allowed_fields
+StatsBase.describe(df::AbstractDataFrame) = describe(df, :mean, :min, :median, 
+                                                     :max, :nunique, :nmissing, 
+                                                     :eltype)
+function StatsBase.describe(df::AbstractDataFrame, stats::Union{Symbol, Pair}...)
+   
+    predefined_funs = Symbol[s for s in stats if s isa Symbol] 
+    if :all in predefined_funs
```

Comment:
I think that when passing `:all` we should assume that it is the only `Symbol`-type of argument.